scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/scheme/TypedParquetTupleScheme.scala

```diff
@@ -95,6 +79,45 @@ trait ParquetWriteSupport[T] extends WriteSupport[T] {
   def writeRecord(r: T, rc: RecordConsumer, schema: MessageType): Unit
 }
 
+object ParquetInputOutputFormat {
+  val READ_SUPPORT_INSTANCE = "parquet.read.support.instance"
```

Comment:
maybe this should be `scalding.parquet.read.support.instance` to be sure we don't collide with something from Parquet.